google/cloud/storage/oauth2/service_account_credentials.h

```diff
@@ -142,6 +143,27 @@ class ServiceAccountCredentials : public Credentials {
         status, status.ok() ? authorization_header_ : std::string(""));
   }
 
+  /**
+   * Sign a blob using the credentials.
+   *
+   * Create a RSA SHA256 signature of the blob using the Credential object. If
+   * the credentials do not support signing blobs it returns an error status.
+   *
+   * @param blob the bytes to sign.
+   * @return a Base64-encoded RSA SHA256 digest of @p blob using the current
+   *   credentials.
+   */
+  std::pair<google::cloud::storage::Status, std::string> SignBlob(
```

Comment:
Just one more nit: For consistency, can we name this either `SignBytes` (after the method in the Python auth library [1]), or `SignString`, which would line up with our Signed URL method `StringToSign` and our OpenSslUtils method `SignStringWithPem`?  Since we already use "String" elsewhere, I'd suppose I'd prefer the latter, but I don't feel strongly.

[1] https://github.com/googleapis/google-auth-library-python/blob/9cdfaaaf5827cbc3c96126e0a2605eba62c3eb89/google/auth/credentials.py#L297

Reply:
Done.